server/channels/api4/post_test.go

```diff
@@ -3193,6 +3229,7 @@ func TestGetEditHistoryForPost(t *testing.T) {
 
 func TestCreatePostNotificationsWithCRT(t *testing.T) {
 	th := Setup(t).InitBasic()
+	defer th.TearDown()
```

Comment:
Never previously torn down. (Ideally, this would just be a `t.Cleanup` inside `Setup`, but I'm deferring that spammier change to another time.)

Reply:
Found some instances of this on the product test refactor too and 100% agree, we should move this to the setup to forget about explicitly tearing things down